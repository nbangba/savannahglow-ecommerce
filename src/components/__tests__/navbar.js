/**
 * @jest-environment jsdom
 */

import React from "react"
import renderer from "react-test-renderer"

import Navbar from "../navbar"


describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Navbar />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})