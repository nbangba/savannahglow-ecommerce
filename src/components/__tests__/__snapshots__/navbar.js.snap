// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="sc-furwcr gnHaag"
>
  <div
    className="sc-egiyK jLllpU"
    onClick={[Function]}
  >
    <test-file-stub
      style={
        Object {
          "height": 40,
          "width": 40,
          "zIndex": 100,
        }
      }
    />
  </div>
  <div
    className="sc-iCfMLu jSVSGn"
  />
  <div
    className="logoMobile"
  >
    <test-file-stub
      style={
        Object {
          "height": 80,
          "width": 80,
        }
      }
    />
  </div>
  <ul
    className="sc-pVTFL ggZkWJ desktop-menu"
    style={
      Object {
        "flexGrow": 10,
      }
    }
  >
    <li
      className="sc-jrQzAO cZxuGg"
    >
      <a
        href="/"
      >
        Home
      </a>
       
    </li>
    <li
      className="sc-jrQzAO cZxuGg"
    >
      <a
        href="/blogs"
      >
        Blogs
      </a>
       
    </li>
    <li
      className="sc-jrQzAO cZxuGg"
    >
      <a
        href="/faq"
      >
        Faq
      </a>
       
    </li>
    <li
      className="sc-jrQzAO cZxuGg"
    >
      <a
        href="/products"
      >
        Products
      </a>
       
    </li>
  </ul>
  <div
    role="alert"
  >
    <p>
      Error loading projects:
    </p>
    <pre>
      SDK not found. useSdk must be called from within a provider
    </pre>
  </div>
</div>
`;
